### DISCUSSION TITLE
Shortcuts in other Blockly applications (i.e. Scratch) apply to blocks in different ways

### DISCUSSION THREAD

**Discussion by BeksOmega** [RaspberryPiFoundation/blockly-keyboard-experimentation#21] [General]
Blockly allows you to change the behavior of keyboard shortcuts. By default in Blockly (and in MakeCode) actions like "copy" and "delete" only apply to the selected block and inner children. While in scratch "copy" copies a block and all **next** children, and "delete" only applies to the selected block and inner children.

![Screen recording 2024-06-27 1 01 45 PM](https://github.com/google/blockly-keyboard-experimentation/assets/25440652/ec3d30f0-9828-4ff5-84dd-dc1425e97e94)
![Screen recording 2024-06-27 1 03 10 PM](https://github.com/google/blockly-keyboard-experimentation/assets/25440652/cc889838-d2ea-4a85-b86c-3ef7cba776ee)

This creates inconsistency among the different applications. It also makes it difficult (in the scratch case) to indicate visually or auditorily what blocks will be affected before you perform an action. Is it all the next blocks? or only the selected block and its inner children?

Should we continue supporting this behavior? Or should we require applications to be more consistent?

**Comment by rachel-fenichel** [comment]
@ericrosenbaum we'd like to get a Scratch opinion on this behaviour. Are you the right person to tag? And if not, who should we tag in to discuss?
